### DISCUSSION TITLE
Help with updating all my branches?

### DISCUSSION THREAD

**Discussion by icopycodelol** [community/community#40284] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi im new to github can sm1 tell me how do you get the updated versions all the branches of a git repository?
git fetch isn't doing anything

**Comment by ashwinn-v** [comment, marked as answer]
I believe you want the latest version on your local branches from remote? Then a simple `git pull --all` should work for you
Check this link for more info `https://stackoverflow.com/questions/10312521/how-do-i-fetch-all-git-branches`

**Comment by white10146** [comment]
Thanks
